packages/odie-client/src/components/message/user-message.tsx

```diff
@@ -68,7 +76,9 @@ export const UserMessage = ( {
 				<WasThisHelpfulButtons message={ message } isDisliked={ isDisliked } />
 			) }
 
-			{ ! showExtraContactOptions && <DirectEscalationLink messageId={ message.message_id } /> }
+			{ ! showExtraContactOptions && ! isExperimentGiveWapuuAChance && (
+				<DirectEscalationLink messageId={ message.message_id } />
+			) }
```

Comment:
Can you experiment reverting this change? Removing `&& ! isExperimentGiveWapuuAChance` should continue to work for both situations since you are already setting `showExtraContactOptions` correcting for both scenarios.

Reply:
Removing it adds the direct escalation link for any question. If you think is ok, I revert that change
![image](https://github.com/user-attachments/assets/302fa742-382f-4235-967b-0b2b9c969edd)

Reply:
I think the behavior shouldn't change. Let's merge it as it is.